docs/reference/treyresearch/armTemplates/auxiliary/mgmtGroups.json

```diff
@@ -5,7 +5,28 @@
         "topLevelManagementGroupPrefix": {
             "type": "string",
             "metadata": {
-                "description": "Provide suffix for the management group structure."
+                "description": "Provide prefix for the management group structure."
+            }
+        },
+        "platformMgs": {
+            "type": "array",
+            "defaultValue": [
+                "management",
+                "connectivity",
+                "identity"
+            ],
+            "metadata": {
+                "description": "Management groups for platform specific purposes, such as management, networking, identity etc."
+            }
+        },
+        "landingZoneMgs": {
+            "type": "array",
```

Comment:
Per earlier discussions, are you also starting with both Online and Corp, or only landingzone MG for the workload?

Reply:
We were thinking of landingzone and Production and Non-Prod underneath, but wanted to have a chat with you around that to hear from you around that vs Online/Corp.

Reply:
Pls have same consistent pattern as the other reference implementation. As principle, the test/dev/prod partitioning is not going into the overall architecture, but allow application scenarios to have their own separation within the landing zones themselves.